### PR TITLE
Relicensing to LGPL v3+

### PR DESCRIPTION
**Pull request by fridex**
Depends-On: #67

### CODE REVIEW

**Comment by fridex** [comment]
I'll wait for #67 to add respective changes to setup.py.

**Comment by frenzymadness** [comment]
The header in .py files should be modified as well.

**Comment by fridex** [comment]
Following https://www.gnu.org/licenses/gpl-howto.en.html, the "lesser" part of GPL is on top of GPL so we should include GPL and LGPL files (COPYING.LESSER) in the repo.

**Comment by fridex** [comment]
> Following https://www.gnu.org/licenses/gpl-howto.en.html, the "lesser" part of GPL is on top of GPL so we should include GPL and LGPL files (COPYING.LESSER) in the repo.

@frenzymadness do you want to have the ``COPYING.LESSER`` file stated in ``MANIFEST.in`` for the RPM release?

Also, I don't know github conventions here, [but it looks like they want ``LICENSE`` file](https://help.github.com/en/github/building-a-strong-community/adding-a-license-to-a-repository) - Do we want to keep the ``LICENSE`` file or rename it to ``COPYING``? Or the other way around - rename ``COPYING.LESSER`` to ``LICENSE.LESSER``? Any project already packaged as RPM to learn from?

**Comment by fridex** [comment]
Discarding the WIP flag, ready to be reviewed -- considering the questions I asked above to match conventions.

**Comment by frenzymadness** [comment]
Tons of Rust projects have two licenses (MIT and Apache) so we can take inspiration from them. I'd prefer to have two files `LICENSE-GPL` and `LICENSE-LGPL`, include them both in Manifest so I can have them in RPM package and I'll also include both licenses in RPM headers. What do you think?

**Comment by fridex** [comment]
> Tons of Rust projects have two licenses (MIT and Apache) so we can take inspiration from them. I'd prefer to have two files `LICENSE-GPL` and `LICENSE-LGPL`, include them both in Manifest so I can have them in RPM package and I'll also include both licenses in RPM headers. What do you think?

Sounds good to me. I've also kept ``LICENSE`` as a simpling to ``LICENSE-LGPL`` so the license info is obtainable/detect by GitHub (let's see if it will work out). PTAL

File: setup.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ def get_version():
     classifiers=[
         "Development Status :: 4 - Beta",
         "Intended Audience :: Developers",
-        "License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)",
+        "License :: OSI Approved :: GNU Lesser General Public License v3 or later (GPLv3+)",
```
Comment (frenzymadness) [inline setup.py:42]:
```suggestion
        "License :: OSI Approved :: GNU Lesser General Public License v3 or later (LGPLv3+)",
```

**Pull request merged**